styles.css

```diff
@@ -1 +1,89 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.grid-container  {
+    background-color: hsl(33, 100%, 93%);
+    color: hsl(0, 0%, 100%);
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-column-gap: 16px;
+    grid-row-gap: 16px;
+    grid-template-areas:
+        "A"
+        "B"
+        "C"
+        "D"
+        "E"
+        "."
+        "F"
+}
+
+.item-1, .item-2, .item-3, .item-4, .item-5, .item-6  {
```

Comment:
nette code
Hier heb ik alle code geplaatst die voor alle items hetzelfde is. Dus ook de flex instellingen.
Daarna heb ik, per item, de afwijkende code geplaatst. Scheelt copy-paste als er iets vervangen moet worden.
jouw hsl levert extra bonus-punten op, die mis ik in mijn code :-(